system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -262,3 +261,104 @@
     }
   }
 }
+
+.topsite-form {
+  $input-border-color: #424F59;
+  $error-background-color: #D63920;
+  $error-shadow-color: rgba(231, 111, 92, 0.75);
+
+  .form-wrapper {
+    padding: 15px 0;
+
+    .field {
+      margin-inline-start: 205px;
+      position: relative;
+    }
+
+    .section-title {
+      margin-bottom: 5px;
+      margin-inline-start: 205px;
+    }
+
+    input {
+      &[type='text'],
+      &[type='email'] {
+        border: solid 1px $input-border-color;
+        border-radius: 2px;
+        margin: 5px 0;
+        padding: 7px;
+        width: 350px;
+
+        &::placeholder {
+          font-style: italic;
+        }
+      }
+    }
+
+    .invalid {
+      input {
+        &[type='text'],
+        &[type='email'] {
+          border: solid 1px $error-background-color;
+          box-shadow: 0 0 0 2px $error-shadow-color;
+        }
+      }
+    }
+
+    .error-tooltip {
+      animation: fade-up-tt 450ms;
+      background: $error-background-color;
+      border-radius: 2px;
+      color: $white;
+      offset-inline-start: 3px;
+      padding: 5px 12px;
+      position: absolute;
+      top: 44px;
+      z-index: 1;
+
+      // tooltip caret
+      &::before {
+        background: $error-background-color;
+        bottom: -8px;
+        content: '.';
+        height: 16px;
+        offset-inline-start: 12px;
+        position: absolute;
+        text-indent: -999px;
+        top: -7px;
+        transform: rotate(45deg);
+        white-space: nowrap;
+        width: 16px;
+        z-index: -1;
+      }
+    }
+  }
+
+  .actions {
+    justify-content: flex-end;
+
+    button {
+      margin-inline-start: 10px;
+      margin-inline-end: 0;
+
+      &.add,
+      &.save {
+        background: $input-primary;
+        border: solid 1px $blue-60;
+        color: $white;
```

Comment:
Can these buttons just also have `.done` classname?

Reply:
yeah, i guess we could add the `done` class to them. But I use the `.save` and `.add` for testing purposes. Maybe we should have the class for styling be `.primary` instead? Then we would have "primary done", "primary add" and "primary save".

Reply:
You can keep `save` and `add` to distinguish them. Just also add `done` to get the styling.

Reply:
👍 fixord